### PR TITLE
Update feed on changes

### PR DESCRIPTION
**Pull request by tsauerwein**
The feed is updated...

* When a new document is created (no user or image).
* When a new user is associated to an outing.
* When a group of images is uploaded.

Related to https://github.com/c2corg/v6_api/issues/403

### CODE REVIEW

File: c2corg_api/models/feed.py [hunk 1]
```diff
@@ -367,6 +401,53 @@ def update_feed_association_update(
     update_participants_of_outing(child_document_id, user_id)
 
 
+def update_feed_images_upload(images, images_in, user_id):
```
Comment (asaunier) [inline c2corg_api/models/feed.py]: What's the difference between ``images`` and ``images_in``?

File: c2corg_api/views/feed.py [hunk 2]
```diff
@@ -276,11 +276,18 @@ def load_feed(changes, lang):
...
 
-    # TODO deal with redirected documents
-
     documents_to_load = get_documents_to_load(changes)
     documents = load_documents(documents_to_load, lang)
 
+    # only return changes for which the document/user could be loaded
+    changes = [
+        c for c in changes
+        if documents.get(c.user_id) and documents.get(c.document_id)
```
Comment (asaunier) [inline c2corg_api/views/feed.py]: Not sure to understand how this is related to redirected documents?

**Comment by asaunier** [comment]
Looks very good to me, thanks!

Just 2 comments/questions:
* is this ``filter=0`` parameter really needed to disable filters in the personal feed? Why not simply using the standard feed instead?
* is the ``quality`` attribute taken into account when adding documents to the feed? (it seems not :P)

**Comment by tsauerwein** [comment]
> is this filter=0 parameter really needed to disable filters in the personal feed? Why not simply using the standard feed instead?

Indeed. If that works for you on the client-side, I will remove the parameter.

> is the quality attribute taken into account when adding documents to the feed? (it seems not :P)

It wasn't very clear what you wanted. We have to discuss this again.

File: c2corg_api/models/feed.py (hunk 1 above)
Comment (tsauerwein) [inline c2corg_api/models/feed.py]: `images` are image entities, `images_in` is what was received in the request data. This naming schema `document/document_in` is also used elsewhere.

File: c2corg_api/views/feed.py (hunk 2 above)
Comment (tsauerwein) [inline c2corg_api/views/feed.py]: Redirected documents are not included in the response to `load_documents` (one line above).

**Comment by asaunier** [comment]
> Indeed. If that works for you on the client-side, I will remove the parameter.

I don't know. I have added a comment in the UI PR: https://github.com/c2corg/v6_ui/pull/731#issuecomment-254745091
@ginold what do you think?
Is it more convenient to add a parameter to the request or to switch the URL?

> It wasn't very clear what you wanted. We have to discuss this again.

OK I will create a dedicated issue to discuss this part (no big deal if not ready for the beta). It can probably be handled in a separate PR, can't it?

**Pull request merged**